.github/workflows/lint-docs.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
+          use-quiet-mode: 'yes'
```

Comment:
/hold

this is adding the line to the setup-node action, not the markdown checker itself

Reply:
it seems we are using a different lint checker than CAPI. To align we could replace the entire jobs section with:

```
jobs:
  markdown-link-check:
    name: Broken Links
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - uses: gaurav-nelson/github-action-markdown-link-check@v1
      with:
        use-quiet-mode: 'yes'
        config-file: .markdownlinkcheck.json
```

I realize this is a bigger scope than initially advertised and this was a good-first-issue so if you don't have bandwidth feel free to close this PR and someone else can take over.

Reply:
Sorry about the confusion, I made the changes. If you have any other changes to make, let me know.

Reply:
Could you please push a fake commit to change a doc file (*.md) so we can make sure that this runs properly? The Github Action is only triggered on PRs that contain doc changes

Reply:
Done ;)